### DISCUSSION TITLE
no image preview or updated preview after installation

### DISCUSSION THREAD

**Discussion by Disincentivato** [fuatakgun/eufy_security#1030] [Q&A]
After a new installation I manage to have all my 6 devices on the integration. I have 2 p&t camera, 2 S221 camera and one 2C camera. All this camera are listed as TB410. I have all the preview from the installation time, no update image. In one camera no preview available. Any idea?
![Immagine 2023-12-05 080128](https://github.com/fuatakgun/eufy_security/assets/90531670/22788263-8c38-483c-8358-771d26df1cf8)

**Comment by fuatakgun** [comment]
Did you enable all motion notifications for your account so images can be updated?

Non image camera, does it onboard memory or requires an SD card?

**Reply by Disincentivato** [reply]
Hi thanks for your reply. I actually had an SD card error on that camera. I formatted it and it seems to be ok.
However, the image does not yet appear on HA. I don't even have updated pictures of all the other cams. They all record on internal SD.

**Reply by Disincentivato** [reply]
I had not enabled motion notification on the security panel! I thought it was enough to just enable it from the camera settings. Thanks, all solved!
